packages/react-examples/src/react-charting/SankeyChart/SankeyChart.Rebalance.Example.tsx

```diff
@@ -0,0 +1,95 @@
+import { IChartProps, ISankeyChartProps, SankeyChart } from '@fluentui/react-charting';
+import * as React from 'react';
+
+interface ISankeyChartBasicState {
+  width: number;
+  height: number;
+}
+
+export class SankeyChartRebalanceExample extends React.Component<{}, ISankeyChartBasicState> {
+  constructor(props: ISankeyChartProps) {
+    super(props);
+    this.state = {
+      width: 820,
+      height: 400,
+    };
+  }
```

Comment:
ToDo: Not a PR blocker.
Could you update the data with a more involved real world dataset so that the users can understand the intricacies.
I believe your team is one of the power users of sankey diagram and testing its limits.

Reply:
I'm not sure what you are looking for, here. This example was a minimum viable test to reproduce the issue and help me ensure that the fix was accurate.
We have real-world data in https://github.com/microsoft/fluentui/pull/30448, but I think that one may not trigger this issue. I can get different data which does trigger this issue, though.

Reply:
The example in #30448 should be good. I was just meaning to add a more real world example where the sankey implementation can be stress tested, not specific to this PR.